java/Nikiforov-Oleksandr/src/com/zoolatech/lecture3/tasks/_4/CsvReporter.java

```diff
@@ -0,0 +1,20 @@
+package com.zoolatech.lecture3.tasks._4;
+
+public class CsvReporter extends Reporter {
+
+    public CsvReporter(String headerData, String fileData, String fileName) {
+        super(headerData, fileData, fileName);
+    }
+
+    @Override
+    void generate() {
```

Comment:
methods `generate` in children classes look pretty similar, try to remove the duplicated code